tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "esModuleInterop": true,
     "sourceMap": true,
     "declaration": false,
-    "downlevelIteration": true,
     "experimentalDecorators": true,
```

Comment:
We could drop `experimentalDecorators` too, as we aren't using `@Inject` in this project.